src/webots/nodes/WbReceiver.cpp

```diff
@@ -117,7 +117,7 @@ WbReceiver::WbReceiver(WbTokenizer *tokenizer) : WbSolidDevice("Receiver", token
   init();
 }
 
-WbReceiver::WbReceiver(const WbReceiver &other) : WbSolidDevice(other) {
+WbReceiver::WbReceiver(const WbReceiver &other) : WbSolidDevice(other), mTransmissionList(), mWaitingQueue(), mReadyQueue() {
```

Comment:
The newly added initialization instructions are useless and should be removed.

Reply:
cppcheck triggers an error if they are missing.